### PR TITLE
T07- Chess board.

### PR DESCRIPTION
**Pull request by Priyanka15-12**

### CODE REVIEW

**Comment by MetaNaveen** [comment]
Got alignment issue.

screenshot_20230822_010123_01

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+using System.Text;
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
```
Comment (MetaNaveen) [inline Program.cs:5]: Fix the alignment issue mentioned in the Conversation.

**Review by MetaNaveen — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 1 above)
Comment (Priyanka15-12) [reply Program.cs:5]: Done

**Review by MetaNaveen — APPROVED** [review]

**Comment by MetaNaveen** [comment]
Working as expected as per the image given in doc. (But the positioning of white/black king is incorrect)

**Comment by Priyanka15-12** [comment]
Changed the position of king.

**Pull request closed without merging**